arrow-core-test/build.gradle

```diff
@@ -16,6 +16,6 @@ dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
     compile "org.jetbrains.kotlinx:kotlinx-coroutines-core:$KOTLINX_COROUTINES_VERSION"
     testRuntime "org.junit.vintage:junit-vintage-engine:$JUNIT_VINTAGE_VERSION"
-    compile "io.kotlintest:kotlintest-runner-junit5:$KOTLIN_TEST_VERSION", excludeArrow
+    compile "io.kotest:kotest-runner-junit5-jvm:$KOTLIN_TEST_VERSION", excludeArrow
```

Comment:
I'd just upgrade `arrow-core-test` which is using KotlinTest to **compile**. 

I think all the necessary dependencies should be:

```   
    compile "io.kotest:kotest-runner-junit5-jvm:$KOTEST_VERSION" 
    compile "io.kotest:kotest-assertions-core-jvm:$KOTEST_VERSION"
    compile "io.kotest:kotest-property-jvm:$KOTEST_VERSION"
    compile "io.kotest:kotest-runner-console-jvm:$KOTEST_VERSION"`
```

`excludeArrow` is not necessary because **Kotest** doesn't show Arrow as a transitive dependency.